src/engraving/style/textstyle.cpp

```diff
@@ -1019,49 +1019,49 @@ static constexpr std::array<TextStyleName, int(Tid::TEXT_STYLES)> textStyles { {
     { QT_TRANSLATE_NOOP("TextStyle", "Lyricist"),                &lyricistTextStyle,          Tid::POET },
     { QT_TRANSLATE_NOOP("TextStyle", "Translator"),              &translatorTextStyle,        Tid::TRANSLATOR },
     { QT_TRANSLATE_NOOP("TextStyle", "Frame"),                   &frameTextStyle,             Tid::FRAME },
-    { QT_TRANSLATE_NOOP("TextStyle", "Instrument Name (Part)"),  &partInstrumentTextStyle,    Tid::INSTRUMENT_EXCERPT },
```

Comment:
so changing the casing of these localized text style names actually broke one of the tests, because it tries to find "Instrument Name (Part)" as a style...but localized strings shouldn't be getting used as style identifiers?